src/node_crypto.cc

```diff
@@ -847,29 +847,43 @@ static unsigned long AddCertsFromFile(  // NOLINT(runtime/int)
   return err;
 }
 
+
+static void IsRootCertStoreEmpty(const FunctionCallbackInfo<Value>& args) {
+  return args.GetReturnValue().Set(
+    Boolean::New(args.GetIsolate(), !root_cert_store));
```

Comment:
`Set()` has an overload that accepts a bool, no need to wrap it in a `v8::Boolean`.

Style issue: use `root_cert_store == nullptr`, not `!root_cert_store`.

Reply:
Done.